test/sql/window/test_streaming_window.test

```diff
@@ -131,3 +131,33 @@ query TT
 explain select last_value(i) over (order by j), first_value(i) over () from integers
 ----
 physical_plan	<REGEX>:.*STREAMING_WINDOW.*WINDOW.*
+
+
+statement ok
+CREATE TABLE v1(id bigint);
```

Comment:
Could we add some tests where we use the streaming window operator in:

* Recursive CTEs
* In a prepared statement that is executed multiple times